int32 episode_number
int32 robot_id
float32 episode_reward
